### PR TITLE
New content manager schemas + wysiwyg type

### PR DESCRIPTION
**Pull request by alexandrebodin**
#### Description of what you did:

New content manager schemas + wysiwyg type

#### My PR is a:

- [ ] 💥 Breaking change
- [ ] 🐛 Bug fix
- [ ] 💅 Enhancement
- [ ] 🚀 New feature

#### Main update on the:

- [ ] Admin
- [ ] Documentation
- [ ] Framework
- [ ] Plugin

#### Manual testing done on the following databases:

- [ ] Not applicable
- [ ] MongoDB
- [ ] MySQL
- [ ] Postgres
- [ ] SQLite

### CODE REVIEW

**Comment by derrickmehaffy** [comment]
For clarification this is creating a new field type instead of storing WYSIWYG in the content-manager right?

**Comment by alexandrebodin** [comment]
> For clarification this is creating a new field type instead of storing WYSIWYG in the content-manager right?

One of the commits does that :) The rest just changes completely the format of the content manager schema storage

**Comment by derrickmehaffy** [comment]
> For clarification this is creating a new field type instead of storing WYSIWYG in the content-manager right?

Cool that's what I thought, just wanted to check to make sure I wasn't reading into things :wink:

File: packages/strapi-plugin-content-manager/services/utils/configuration/metadatas.js [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+  };
+}
+
+function createDefaultMetadata(model, name) {
+  const attr = model.allAttributes[name];
+  const edit = {
+    label: name,
+    description: '',
+    placeholder: '',
+    visible: true, // TODO: depends on the type ?
```
Comment (lauriejim) [inline packages/strapi-plugin-content-manager/services/utils/configuration/metadatas.js]: By default, all types are visible in the Edit View

File: packages/strapi-plugin-users-permissions/config/layout.js [hunk 2]
```diff
@@ -6,19 +6,5 @@ module.exports = {
       destroy: 'User.destroy',
       deleteall: 'User.destroyAll',
     },
-    attributes: {
```
Comment (lauriejim) [inline packages/strapi-plugin-users-permissions/config/layout.js]: Is it normal to remove the default User layout?

Comment (alexandrebodin) [reply packages/strapi-plugin-users-permissions/config/layout.js]: We are not going to use thoses classes anymore. We can define a new way to have default  layouts though. We didn't really talk about that. @Aurelsicoko @soupette might need to check that

**Pull request merged**